crypto/hpke/hpke_highlevel.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* An OpenSSL-based high level interface to HPKE */
+#include <stddef.h>
+#include <openssl/hpke.h>
+#include <openssl/core_names.h>
+#include <openssl/crypto.h>
+#include <internal/packet.h>
+#include <openssl/err.h>
+
+int OSSL_HPKE_seal_base(OSSL_HPKE_SUITE suite,
+						const unsigned char *receiver, size_t receiverlen,
```

Comment:
indentation is still wonky.. Should look like this in the editor here.. (No tabs please) 
````
int OSSL_HPKE_seal_base(OSSL_HPKE_SUITE suite,
                                             const unsigned char *receiver, size_t receiverlen,
````

Reply:
This is caused by tabs.  I suppose that the author has the setting tab=4 columns.  Github shows them with the setting tab=8 columns

Reply:
Thank you, that explains the mysteries of the indentation errors I am seeing.

Perhaps CI could include a no-tabs check to make this easier to discover?